retrace/glretrace.py

```diff
@@ -171,9 +171,38 @@ def retraceFunctionBody(self, function):
             print(r'    if (currentContext && currentContext->features().pixel_buffer_object) {')
             print(r'        glGetIntegerv(GL_PIXEL_PACK_BUFFER_BINDING, &_pack_buffer);')
             print(r'    }')
-            print(r'    if (!_pack_buffer) {')
+            print(r'    if (!_pack_buffer && !retrace::runReadback) {')
             print(r'        return;')
             print(r'    }')
+            print(r'    if (retrace::runReadback) {')
```

Comment:
What if `_pack_buffer` is true here?  It would be wrong to do this.

Reply:
Should be fixed with the refactoring.